ArduCopter/RC_Channel.cpp

```diff
@@ -252,8 +252,10 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
 #if RANGEFINDER_ENABLED == ENABLED
             if ((ch_flag == HIGH) && copter.rangefinder.has_orientation(ROTATION_PITCH_270)) {
                 copter.rangefinder_state.enabled = true;
+                gcs().send_text(MAV_SEVERITY_WARNING, "SurfaceTracking enabled");
```

Comment:
using rangefinder maybe ? As this insn't only for surface tracking (landing, precland, etc.)

Reply:
originally thought of that , but this is only for the downward facing,ground/sruface tracking rangefinder....and you actually are tracking the surface during landing,etc.....but open to anything better that doesn't imply other rangefinders...